okapi-core/src/main/java/org/folio/okapi/managers/ModuleManager.java

```diff
@@ -38,6 +38,7 @@
   private String mapName = "modules";
   private LockedTypedMap1<ModuleDescriptor> modules
     = new LockedTypedMap1<>(ModuleDescriptor.class);
+  private Map<String,ModuleDescriptor> enabledModules = new HashMap<>();
```

Comment:
Is the cache pruned when ModuleManager.update or ModuleManager.delete is called, and are there unit tests for update and delete?

Reply:
In reality not needed, but I'll just do it. MDs are immutable and probably MD PUT will be removed soon.